api/tsconfig.json

```diff
@@ -16,7 +16,13 @@
         "strictNullChecks": true
     },
     "files": [
+        "api.ts",
+        "bucket.ts",
         "index.ts",
+        "service.ts",
+        "table.ts",
+        "timer.ts",
+        "topic.ts",
         "types.ts"
```

Comment:
i'm surprised these aren't pulled in by referencing 'index'. Was this necessary...?  What impact does this have?

Reply:
> What impact does this have?

This is needed for documentation generation to correctly find all the exported APIs in @pulumi/cloud.

This was needed to fix https://github.com/pulumi/docs/pull/536 (I just didn't check in the change here when I made that fix).  If we don't check this in, the next attempt to update the API docs will remove those docs.

May well be a bug in typedoc - but pending an alternate workaround I think we need to do this.

Reply:
Ah, good to know.  Thanks!